### ISSUE TITLE
Unittest long_double_representation raises ValueError

### ISSUE THREAD

**Issue by erikjoh** [nicklashersen/numpy#50]
Just create a unittest that sends an empty list as parameter. That should cause this branch https://codecov.io/gh/numpy/numpy/src/master/numpy/core/setup_common.py#L401 to be run.
Assert that the ValueError is raised. Super easy I think.

**Comment by Eika96** [comment]
Just create the file `numpy/core/tests/test_setup_common.py` and add something like:

```python
from __future__ import division, absolute_import, print_function

import pytest

from numpy.core.setup_common import long_double_representation
from numpy.testing import assert_

class TestSetupCommon(object):

    def test_long_double_representation_value_error(self):
        """
        Test that an invalid input gives a correct ValueError.
        """

        with pytest.raises(ValueError) as excinfo:   
            long_double_representation([])

        assert_(str(excinfo.value) == 'Could not lock sequences (None)')
```

**Comment by erikjoh** [comment]
Fixed in #81
